README.md

```diff
@@ -70,10 +71,11 @@ For clarity, an example of the setup below can be found in the [ENS subgraph rep
     {
       "scripts": {
         "codegen": "graph codegen --output-dir types/",
+        "create-subgraph": "graph create <SUBGRAPH-NAME> --node http://127.0.0.1:8020",
```

Comment:
Shall we call this `create-local`, since it's only for local environments?

Let's drop `build`, `build-ipfs` as well.

Reply:
Actually, make it `create` instead of `create-local` please.